main.c

```diff
@@ -15,11 +15,18 @@
 #define WHITE_LED_OFF (LATC6 = 1)
 #define BLUE_LED_ON (LATC7 = 0)
 #define BLUE_LED_OFF (LATC7 = 1)
+#define GET_BATTERY_VOLTAGE ADCC_GetSingleConversion(RA0)
+#define GET_CURRENT_DRAW ADCC_GetSingleConversion(RA1)
+#define GET_BATTERY_VOLTAGE ADCC_GetSingleConversion(RB5)
 #define HIGH_DAC_ADDRESS 0x4d
 #define LOW_DAC_ADDRESS 0x4c
+#define HIGH_VAL 50
+#define LOW_VAL 3
+
+int BatV = 0;
```

Comment:
does this need to be a global variable?

Reply:
That is just so I can view it while debugging to make sure I was reading the analog signal correctly